src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+
+  @DisplayName("size를 사용하여 Set의 크기를 확인한다.")
+  @Test
+  void requirement1(){
+    assertThat(numbers.size()).isEqualTo(3);
```

Comment:
컬렉션 사이즈를 검증하는 메소드를 사용해보면 어떨까요?
```suggestion
    assertThat(numbers).hasSize(3);
```

Reply:
assertj에 대해서 공부를 더 해봐야겠네요..! 감사합니다!